web/src/containers/BudgetSummary.js

```diff
@@ -12,25 +12,24 @@ const categoryLookup = {
   combined: 'Subtotal'
 };
 
-function DataRow({ data, title, year }) {
+function DataRow({ data, title }) {
   return (
-    <tr>
-      <th headers="summary-budget-null1 summary-budget-null2">{title}</th>
+    <tr
+      className={ title===categoryLookup.combined ? 'budget-table--subtotal budget-table--row__highlight' : '' }
+    >
+      <th scope="row">{title}</th>
```

Comment:
For these simpler tables without double decker headers, we can forgo the complicated `headers` business in favor of simpler `scope`.